force-app/main/default/triggers/MyFirstTrigger.trigger

```diff
@@ -0,0 +1,3 @@
+trigger MyFirstTrigger on Opportunity (before insert, after insert) {    
+    System.debug('Hello World!');
+}
```

Comment:
I am sure you left this behind by mistake but you never want multiple triggers on an object